### PR TITLE
Add Ranking Component

### LINKED ISSUE #23: UI Komponent | Ranking graczy dla trybu gry
From CodersCamp2020.Project.JavaScript.StarWarsQuiz created by [nowakprojects](https://github.com/nowakprojects): CodersCamp2020/CodersCamp2020.Project.JavaScript.StarWarsQuiz#7  

### Kryteria akceptacji
**Należy zweryfikować i skonsultować, a także ewentualnie uaktualnić przed rozpoczęciem prac nad zgłoszeniem.**

- [x] Komponent zdefiniowany w odrębnym pliku .js
- [x] Komponent przyjmuje na wejściu posortowaną listę 3 najlepszych wyników dla danego trybu gry. Wynik zawiera imię gracza oraz punktację. 
- [x] Komponent wyświetla przekazaną listę rankingową.
- [ ] Działanie komponentu zostało pokryte testami jednostkowymi.
- [x] Wygląd komponentu wg. projektu:
Ranking Title

### PR DESCRIPTION
**Pull request by kamack38**

### CODE REVIEW

File: src/components/Ranking/Ranking.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  static createRankingList(playerScores) {
+    const rankingList = document.createElement('div');
+
+    let playerRanking = [
+      { place: '1st' },
+      { place: '2nd' },
+      { place: '3rd' },
+    ].reduce((acc, item, index) => {
+      return [...acc, { ...item, ...playerScores[index] }];
```
Comment (kacper-cyra) [inline src/components/Ranking/Ranking.js]: For better performance you can use acc.push({...item, ...playerScores[index]}] as it won't recreate acc object for each iteration

File: src/components/Ranking/Ranking.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    this._contents.appendChild(this.container);
+  }
+
+  connectedCallback() {
+    this.appendChild(this._contents);
+  }
+
+  setUserScores(playerScores) {
+    this.playerScores = playerScores;
```
Comment (kacper-cyra) [inline src/components/Ranking/Ranking.js:35]: Did you test that? I think that after updating values you also need to rebuild (render) whole element

**Review by kacper-cyra — APPROVED** [review]

File: src/components/Ranking/Ranking.js (hunk 1 above)
Comment (kamack38) [reply src/components/Ranking/Ranking.js]: After doing so (`acc.push({ ...item, ...playerScores[index] })`) I get an error: 
```
TypeError: can't access property "push", acc is undefined
```

Comment (kacper-cyra) [reply src/components/Ranking/Ranking.js]: Ok, I guess you have returned the same line. Make it work like this:
```
{
    acc.push({ ...item, ...playerScores[index] });
    return acc
}
```

Comment (kamack38) [reply src/components/Ranking/Ranking.js]: Ok, it works now.

**Pull request merged**